client/app/screens/CreateProfileScreen.js

```diff
@@ -95,7 +96,31 @@ class CreateProfileScreen extends React.Component {
 
   _handleFinishButtonPress() {
     const { saveProfileAndLoginAsync, userProfile } = this.props;
+    let error = false;
+
+    this.userProfileFormOptions.forEach(formOption => {
+      if (userProfile[formOption.item] === undefined ||
+          userProfile[formOption.item] === '') {
+        error = true;
+      }
+    });
+
+    if (error) {
+      Toast.show({
+        text: 'Please Fill in All Fields',
+        buttonText: 'Okay',
+        type: 'danger',
+        duration: 3000,
+        textStyle: { fontFamily: 'pt-sans' },
+      });
+      return;
+    }
+
     saveProfileAndLoginAsync({ ...userProfile, isAuthenticated: true });
+    Toast.show({
+      text: `${userProfile.firstName} Signed In`,
+      textStyle: { fontFamily: 'pt-sans' }
```

Comment:
What do you think about having a toast constant object that we can extend so we don't need to write `textStyle` every time?

Eg: constants.js
```js
export const toastDefaults = { textStyle: { fontFamily: 'pt-sans' } };
```

screen.js:
```js
Toast.show({ ...toastDefaults, text: 'blah' });
```